build.sh

```diff
@@ -74,7 +74,9 @@ initTargetDistroRid()
         if [ "$__BuildOS" == "Linux" ]; then
             if [ ! -e $ROOTFS_DIR/etc/os-release ]; then
                 echo "WARNING: Can not determine runtime id for current distro."
-                export __DistroRid=""
+                #Test hack to generate __DistroRid for Android
+                #TODO Find reliable way to generate __DistroRid for Android.
+                export __DistroRid="android.21-arm64"
```

Comment:
@jyoungyun This may be better; so far I've tried every iteration of detecting `__DistroRid` in this function and all have failed except for `! [[ -n "$__DistroRid" ]]`, which is the same as hardcoding that value.